examples/randlanet_classification.py

```diff
@@ -0,0 +1,255 @@
+import os.path as osp
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from torch.nn import Linear, Sequential
+from tqdm import tqdm
+
+import torch_geometric.transforms as T
+from torch_geometric.datasets import ModelNet
+from torch_geometric.loader import DataLoader
+from torch_geometric.nn import MLP, global_max_pool
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.pool import knn_graph
+from torch_geometric.transforms.base_transform import BaseTransform
+from torch_geometric.utils import softmax
+
+# Default activation, BatchNorm, and resulting MLP used by RandLA-Net authors
+lrelu02_kwargs = {"negative_slope": 0.2}
+
+bn099_kwargs = {"momentum": 0.01, "eps": 1e-6}
+
+
+class SharedMLP(MLP):
+    """SharedMLP following RandLA-Net paper."""
+    def __init__(self, *args, **kwargs):
+        # BN + Act always active even at last layer.
+        kwargs["plain_last"] = False
+        # LeakyRelu with 0.2 slope by default.
+        kwargs["act"] = kwargs.get("act", "LeakyReLU")
+        kwargs["act_kwargs"] = kwargs.get("act_kwargs", lrelu02_kwargs)
+        # BatchNorm with 1 - 0.99 = 0.01 momentum
+        # and 1e-6 eps by defaut (tensorflow momentum != pytorch momentum)
+        kwargs["norm_kwargs"] = kwargs.get("norm_kwargs", bn099_kwargs)
+        super().__init__(*args, **kwargs)
+
+
+class GlobalPooling(torch.nn.Module):
```

Comment:
it looks like the output `pos` and `batch` are never used. Should be fine to drop this module altogether.

Reply:
Removed and replaced with global_max_pool directly.